### PR TITLE
feat: BitVec.[toInt|toFin|getMsbD]_ofBool

### PR DESCRIPTION
**Pull request by tobiasgrosser**
This PR completes the basic API for BitVec.ofBool.

### CODE REVIEW

**Comment by tobiasgrosser** [comment]
changelog-library

File: src/Init/Data/BitVec/Lemmas.lean [hunk 1]
```diff
@@ -241,8 +241,17 @@ theorem toFin_one  : toFin (1 : BitVec w) = 1 := by
 @[simp] theorem toNat_ofBool (b : Bool) : (ofBool b).toNat = b.toNat := by
   cases b <;> rfl
 
+@[simp] theorem toInt_ofBool (b : Bool) : (ofBool b).toInt = Int.bmod b.toInt 2 := by
+  cases b <;> rfl
```
Comment (kim-em) [inline src/Init/Data/BitVec/Lemmas.lean:245]:
```suggestion
@[simp] theorem toInt_ofBool (b : Bool) : (ofBool b).toInt = -b.toInt := by
  cases b <;> rfl
```
It seems unnecessarily complicated to invoke `Int.bmod` here.

File: src/Init/Data/BitVec/Lemmas.lean [hunk 2]
```diff
@@ -241,8 +241,17 @@ theorem toFin_one  : toFin (1 : BitVec w) = 1 := by
...
+
+@[simp] theorem toFin_ofBool (b : Bool) : (ofBool b).toFin = Fin.ofNat' 2 (b.toNat) := by
+  cases b <;> rfl
+
+@[simp] theorem getMsbD_ofBool (b : Bool) : (ofBool b).getMsbD i = (decide (i < 1) && b) := by
```
Comment (kim-em) [inline src/Init/Data/BitVec/Lemmas.lean:250]:
```suggestion
@[simp] theorem getMsbD_ofBool (b : Bool) : (ofBool b).getMsbD i = (decide (i = 0) && b) := by
```
?

**Comment by kim-em** [comment]
Could you rearrange the theorems so that these are adjacent to the other `ofBool` accessors, e.g. `getLsbD_ofBool`?

**Comment by tobiasgrosser** [comment]
awaiting-review

**Pull request merged**